content/applications/sales/sales/products_prices/returns.rst

```diff
@@ -1,73 +1,114 @@
-The :guilabel:`Sales` app provides two different ways to process returns based on whether an
-invoice has been sent or not.
+The Odoo *Sales* app provides two different ways to process returns. The method used depends on
+whether or not an invoice has been sent.
 
 Before invoicing
 ================
 
-Returns are completed using *Reverse Transfers* when a customer decides to return a product before
-an invoice has been sent or validated.
+Returns are completed using *Reverse Transfers* when a customer decides to return a product
+**before** an invoice has been sent or validated.
 
 .. note::
-   In order to use Reverse Transfers, the :guilabel:`Inventory` app must also be installed.
+   In order to use Reverse Transfers, the :guilabel:`Inventory` app **must** be installed.
 
-To start a return, open the :guilabel:`Sales` app, navigate to the customer's sales order, and
-click on the :guilabel:`Delivery` smart button to open the associated delivery order.
+To start a return before invoicing, navigate to the :menuselection:`Sales` app, select the desired
+sales order, and click on the :guilabel:`Delivery` smart button to open the associated delivery order.
 
-On the validated delivery order, click :guilabel:`Return` to open the :guilabel:`Reverse Transfer`
-pop-up window. By default, the :guilabel:`Quantity` matches the validated quantities from the
-delivery order. Update the quantities if necessary. Click on the trash icon next to a line item to
+.. image:: returns/sales-order-delivery-smart-button.png
+   :align: center
+   :alt: A typical sales order with a highlighted delivery smart button in Odoo Sales.
+
+On the validated delivery order, click :guilabel:`Return`.
+
+.. image:: returns/validated-delivery-order-return-button.png
+   :align: center
+   :alt: A validated delivery order with a highlighted Return button in Odoo Sales.
+
+This opens a :guilabel:`Reverse Transfer` pop-up window.
+
+By default, the :guilabel:`Quantity` matches the validated quantities from the delivery order.
+Update the quantities, if necessary. Click on the :guilabel:`🗑️ (trash)` icon next to a line item to
 remove it from the return.
 
 .. image:: returns/reverse-transfer-popup.png
    :align: center
    :alt: The "Reverse Transfer" pop-up window, to make a return before invoicing the customer.
 
 Next, click :guilabel:`Return` to confirm the return. This generates a new warehouse operation for
-the incoming returned product(s). Upon receiving the return, the warehouse team validates the
-warehouse operation. Then, on the original sales order, the :guilabel:`Delivered` quantity will
-reflect the difference between the initial validated quantities and the returned quantities.
+the incoming returned product(s).
+
+.. image:: returns/warehouse-operation-confirmed-return.png
+   :align: center
+   :alt: Warehouse operation after a return has been confirmed in Odoo Sales.
+
+Upon receiving the return, the warehouse team validates the warehouse operation. Then, on the
```

Comment:
Is this done by clicking "Validate" on the order? If so, it would probably be best to specify that using a GUI label.